python/lsst/ts/hvac/xml/hvac_mqtt_to_SAL_XML.py

```diff
@@ -305,8 +305,8 @@ def _parse_limits(limits_string):
     elif re.match(r"^\d$", limits_string):
         lower_limit = 0
         upper_limit = 100
-    else:
-        raise ValueError(f"Couldn't match limits_string {limits_string}")
+    # else:
+    #     raise ValueError(f"Couldn't match limits_string {limits_string}")
```

Comment:
Why this change? Please don't leave commented-out code around (or if you absolutely must, then provide a comment explaining, but it's unsafe because it can easily bit rot)

If you are trying to handle invalid data then can you at least log a warning when this occurs?

Reply:
Thanks for catching this and the next change. I was playing with a new Excel sheet sent to me by DatControl and forgot to create a branch for that and then forgot to revert these changes. But they are essential for the code so I reverted them.